src/app/components/BibPage/components/LinkableField.jsx

```diff
@@ -19,9 +19,17 @@ const LinkableBibField = ({
     ? bibValue.prefLabel || bibValue.label || bibValue.url
     : bibValue;
 
-  const filter = `filters[${field}]=${
-    filterPath ?? bibValue['@id'] ?? bibValue ?? ''
-  }`;
+  let partialUrlEncodedBibValue
+  if (bibValue && typeof bibValue === 'string') {
+    partialUrlEncodedBibValue = bibValue
+      .split('')
+      .map((char) => (char === '&' ? '%26' : char === '+' ? '%2B' : char))
+      .join('');
+  }
+
+  // react router url encoding ignores & and +, so such characters within bibValue need to be encoded manually
+  const filter = `filters[${field}]=${filterPath ?? bibValue['@id'] ?? partialUrlEncodedBibValue ?? bibValue ?? ''
+    }`;
```

Comment:
This generally does encode the `bibValue` if it's a string, but the other options will never be encoded. Perhaps setting a `query` variable and passing that into an encoding could work.

```suggestion
    const queryString = `${filterPath ?? bibValue['@id'] ?? bibValue ?? ''}
    const filter = `filters[${field}]=${encodeURIComponent(queryString)}`;
```